config/Schema.java

```diff
@@ -0,0 +1,108 @@
+package grakn.simulation.config;
+
+import grakn.simulation.agents.AgeUpdateAgent;
+import grakn.simulation.agents.CompanyAgent;
+import grakn.simulation.agents.EmploymentAgent;
+import grakn.simulation.agents.FriendshipAgent;
+import grakn.simulation.agents.MarriageAgent;
+import grakn.simulation.agents.ParentshipAgent;
+import grakn.simulation.agents.PersonBirthAgent;
+import grakn.simulation.agents.ProductAgent;
+import grakn.simulation.agents.RelocationAgent;
+import grakn.simulation.agents.TransactionAgent;
+import grakn.simulation.agents.base.AgentRunner;
+import grakn.simulation.agents.common.CityAgentRunner;
+import grakn.simulation.agents.common.ContinentAgentRunner;
+import grakn.simulation.agents.common.CountryAgentRunner;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Schema {
+
+    static HashMap<String, AgentRunner<?>> AGENTS;
+    static {
+        AGENTS = new HashMap<>();
+        AGENTS.put("MarriageAgent", new CityAgentRunner(MarriageAgent.class));
+        AGENTS.put("PersonBirthAgent", new CityAgentRunner(PersonBirthAgent.class));
+        AGENTS.put("AgeUpdateAgent", new CityAgentRunner(AgeUpdateAgent.class));
+        AGENTS.put("ParentshipAgent", new CityAgentRunner(ParentshipAgent.class));
+        AGENTS.put("RelocationAgent", new CityAgentRunner(RelocationAgent.class));
+        AGENTS.put("CompanyAgent", new CountryAgentRunner(CompanyAgent.class));
+        AGENTS.put("EmploymentAgent", new CityAgentRunner(EmploymentAgent.class));
+        AGENTS.put("ProductAgent", new ContinentAgentRunner(ProductAgent.class));
+        AGENTS.put("TransactionAgent", new CountryAgentRunner(TransactionAgent.class));
+        AGENTS.put("FriendshipAgent", new CityAgentRunner(FriendshipAgent.class));
+    }
+
+    public enum AgentMode {
+        TRACE(true, true),
+        RUN(false, true),
+        OFF(false, false);
+
+        private final Boolean trace;
+        private final Boolean run;
+
+        AgentMode(Boolean trace, Boolean run) {
+            this.trace = trace;
+            this.run = run;
+        }
+
+        public Boolean getTrace() {
+            return trace;
+        }
+
+        public Boolean getRun() {
+            return run;
+        }
+    }
+
+    public enum SamplingFunction {
```

Comment:
I think we should replace this with some expression evaluation library at some point, see JEXL. Not yet though.